pkg/rancher-desktop/backend/__tests__/k3sHelper.spec.ts

```diff
@@ -5,15 +5,14 @@ import os from 'os';
 import path from 'path';
 import util from 'util';
 
+import paths from '@pkg/utils/paths';
```

Comment:
This looks like it's in the wrong location in the import list.

I'm not sure why nothing's complaining about that now though.